app/models/manageiq/providers/openshift/inventory/persister/target_collection.rb

```diff
@@ -0,0 +1,20 @@
+class ManageIQ::Providers::Openshift::Inventory::Persister::TargetCollection < ManageIQ::Providers::Openshift::Inventory::Persister::ContainerManager
+  def targeted
+    false # TODO(lsmola) get ready for true
```

Comment:
what does targeted=true mean?

Reply:
So right now we fetch from the DB much more than we have to. With targeted refresh we would scope the queries with the pods we are touching. But that require more work and I think we will first need DB indexes, to speed up the queries. So probably next release item.

Reply:
if someone looks at this file alone, there is no context and very hard to understand where it fits in.
(I now know after reading the k8s PR)
I wonder if some higher-level comment could help?  though maybe too much in flux now and would rot...